.prettierignore

```diff
@@ -0,0 +1,9 @@
+.cache
+build
+build-module
+build-types
+packages/block-serialization-spec-parser/parser.js
+packages/e2e-tests/plugins
+packages/react-native-editor/bundle
+vendor
+!.*.js
```

Comment:
This one might be not necessary. If I recall correctly it was a way to tell ESLint to validate config files like `.eslintrc.js`. We would have to check.

Reply:
Seems fine without. Removed per 10a6e42c18.